Jenkinsfile

```diff
@@ -54,6 +54,18 @@ def buildManifest = {String manifest, String bitbake_image ->
         sh "vagrant ssh -c \"/vagrant/vagrant-cookbook/yocto/build-images.sh ${yoctoDir} ${bitbake_image}\""
     }
 
+    stage("Copy images and cache ${bitbake_image}") {
+        sh "rm -rf archive"
+        sh "mkdir -p archive"
+        sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/tmp/deploy/images/ /vagrant/archive/\""
+        sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/downloads/ /vagrant/archive/\""
```

Comment:
Is it really necessary to copy the downloads? To my mind they are not needed because you have what you need in the sstate-cache or you want to build something that is not in the sstate-cache and then you would not find it in downloads either.